utils/kata-manager.sh

```diff
@@ -720,26 +721,64 @@ handle_containerd()
 	containerd --version
 }
 
+handle_docker()
+{
+	{ containerd_installed; ret=$?; } || true
+	if [ "$ret" -eq 0 ]
+	then
+		info "Backing up previous containerd configuratinn"
+		local cfg="/etc/containerd/config.toml"
+
+		sudo mv $cfg $cfg.system
```

Comment:
How about appending the date to the backup file too, as we do in `configure_containerd()`? That would mean the script would create 1 file / run, but the timestamps could be useful to help debug issues and I don't think we would need to worry about `ENOSPC`.

Reply:
Done!